int8 right_arm
float64 x0_torso_ll_frame
float64 y0_torso_ll_frame
float64 z0_torso_ll_frame
float64 qx0_torso_ll_frame
float64 qy0_torso_ll_frame
float64 qz0_torso_ll_frame
float64 qw0_torso_ll_frame
float64 kx0_torso_ll_frame
float64 ky0_torso_ll_frame
float64 kz0_torso_ll_frame
float64 kth0_torso_ll_frame
int8 is_force_x0
int8 is_force_y0
int8 is_force_z0
int8 is_force_th0
float64 x1_torso_ll_frame
float64 y1_torso_ll_frame
float64 z1_torso_ll_frame
float64 qx1_torso_ll_frame
float64 qy1_torso_ll_frame
float64 qz1_torso_ll_frame
float64 qw1_torso_ll_frame
float64 kx1_torso_ll_frame
float64 ky1_torso_ll_frame
float64 kz1_torso_ll_frame
float64 kth1_torso_ll_frame
int8 is_force_x1
int8 is_force_y1
int8 is_force_z1
int8 is_force_th1
float64 time
---
int8 status
